script/process_qrc.py

```diff
@@ -23,31 +23,76 @@
 
 """
 
+# Standard library imports
 from __future__ import absolute_import, print_function
-
+from subprocess import call
 import argparse
 import glob
 import os
 import sys
-from subprocess import call
+
+# Third party imports
+from watchdog.events import FileSystemEventHandler
+from watchdog.observers import Observer
+
+# Constants
+HERE = os.path.abspath(os.path.dirname(__file__))
+REPO_ROOT = os.path.dirname(HERE)
+
+
+class QSSFileHandler(FileSystemEventHandler):
+    """QSS File observer."""
+
+    def __init__(self, parser_args):
+        """QSS File observer."""
+        super(QSSFileHandler, self).__init__()
+        self.args = parser_args
+
+    def on_modified(self, event):
+        """Handle file system events."""
+        if event.src_path.endswith('.qss'):
```

Comment:
As mentioned the .qss file is not found in this  path, so the process_qrc is not running, but strangely  the files get their modification date updated.